website/docs/basics/blocks-styles.md

```diff
@@ -0,0 +1,401 @@
+<style>
+  .typography[data-id='210c9bbf733ef5c6aa74c49168ac29a7'] {
+    --typography-color: var(--global-colors-black);
+    --typography-content-align: left;
+  }
+</style>
+
+<div class="typography" data-id="210c9bbf733ef5c6aa74c49168ac29a7">...</div>
+```
+
+This is it for all the magic of CSS variables. Now let us see it in action and all those additional features you can utilize.
+
+## Global variables
+
+Now that we know how CSS variables are outputted we have one additional helper to describe. This helper is loaded in `application-blocks-editor.js` file and it is called `outputCssVariablesGlobal(globalSettings);`.
+It is used to output all CSS variables from the global manifest under the `globalVariables` key to `:root` selector.
+
+> Those variables are available in all your blocks/components.
+
+**Global Manifest:**
+```json
+{
+  "namespace": "eightshift-boilerplate",
+  "background": "#D8262C",
+  "foreground": "#FFFFFF",
+  "globalVariables": {
+    "customBlocksName": "eightshift-block",
+    "maxCols": 12,
+    "breakpoints": {
+      "mobile": 479,
+      "tablet": 1279,
+      "desktop": 1919,
+      "large": 1920
+    },
+    "containers": {
+      "default": "1330px"
+    },
+    "gutters": {
+      "none": "0",
+      "default": "25px",
+      "big": "50px"
+    },
+    "sectionSpacing": {
+      "min":  -300,
+      "max":  300,
+      "step": 10
+    },
+    "sectionInSpacing": {
+      "min":  0,
+      "max":  300,
+      "step": 10
+    },
+    "colors": [
+      {
+        "name": "Infinum",
+        "slug": "infinum",
+        "color": "#D8262C"
+      },
+      {
+        "name": "Black",
+        "slug": "black",
+        "color": "#111111"
+      },
+      {
+        "name": "White",
+        "slug": "white",
+        "color": "#FFFFFF"
+      }
+    ]
+  }
+}
+```
+
+**Output:**
+```html
+<style>
+  :root {
+    --global-custom-blocks-name: eightshift-block;
+    --global-max-cols: 12;
+    --global-breakpoints-mobile: 479;
+    --global-breakpoints-tablet: 1279;
+    --global-breakpoints-desktop: 1919;
+    --global-breakpoints-large: 1920;
+    --global-containers-default: 1330px;
+    --global-gutters-none: 0;
+    --global-gutters-default: 25px;
+    --global-gutters-big: 50px;
+    --global-section-spacing-min: -300;
+    --global-section-spacing-max: 300;
+    --global-section-spacing-step: 10;
+    --global-section-in-spacing-min: 0;
+    --global-section-in-spacing-max: 300;
+    --global-section-in-spacing-step: 10;
+    --global-colors-infinum: #D8262C;
+    --global-colors-black: #111111;
+    --global-colors-white: #FFFFFF;
+  }
+</style>
+```
+
+To use global variable just call:
+```css
+var(--global-colors-infinum);
+```
+
+## Variable Default
+
+As we said in the beginning, all CSS variables are controlled with the block/component manifest but we didn't want to output all attributes as CSS variables (because you don't need them all), so you need to set what attributes you are going to output.
```

Comment:
```suggestion
As we said in the beginning, all CSS variables are defined within the block/component manifest. To avoid outputting all of the variables (because you might not need them all), the attributes that will be output as variables need to be defined.
```